nilearn/_utils/docs.py

```diff
@@ -504,10 +504,17 @@ def custom_function(vertices):
 
 # view
 docdict['view'] = """
-view : {'lateral', 'medial', 'dorsal', 'ventral',\
-        'anterior', 'posterior'}, optional
+view : either a string or pair of floats, optional.
+        If a string, must be in {'lateral', 'medial', 'dorsal', 'ventral',\
+        'anterior', 'posterior'}.
+        If a sequence, must be a pair (elev, azim) of float
+        angles in degrees that will manually set a custom view.
+        E.g., view=[270.0, 90.0] or view=(0.0, -180.0). Note that this
+        manual option is only available when the plotting engine is matplotlib.
+        Otherwise, the view can only be specified via one of the pre-set strings.
```

Comment:
```suggestion
        Otherwise, the view can only be specified via one of the pre-set
        strings.
```

Reply:
Thanks for the fixes - I will have time this week to incorporate the handful of suggestions!